src/w3iProxy/pushProviders/internalPushProvider.ts

```diff
@@ -136,27 +136,31 @@ export default class InternalPushProvider implements W3iPushProvider {
       const clientId = await this.pushClient.core.crypto.getClientId()
       // Retrieving FCM token needs to be client side, outside the service worker.
```

Comment:
Comment is now out of place, needs to be moved

Reply:
Moved comment